nearcore/src/runtime/mod.rs

```diff
@@ -141,6 +141,7 @@ pub struct NightshadeRuntime {
     shard_tracker: ShardTracker,
     genesis_state_roots: Vec<StateRoot>,
     migration_data: Arc<MigrationData>,
+    num_epochs_to_keep_store_data: u64,
```

Comment:
> @mzhangmzz does it make sense that NightshadeRuntime knows when we start GCing epochs? Would something like EpochManager or Client be a better place for this info?

Yeah I think client is probably a better place for this. cc @pmnoxx

Reply:
@matklad We only use `num_epochs_to_keep_store_data` inside `NightshadeRuntime `.

We can put it inside `EpochManager`, I see that nightshade accesses it through `Arc<RwLock<EpochManager>>`. This isn't much of an issue, because we take as look anyway.